RELEASE_NOTES.md

```diff
@@ -6,7 +6,7 @@ The `Timer` now can be started with a delay.
 
 ## Upgrading
 
-<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
+* The `limit` attribute was removed from `Anycast` channel as the same information can be obtained from its buffer through `deque.maxlen` property.
```

Comment:
It wasn't because of #186. Because of the comment I've found it redundant as the variable was only set and actually never used in the channel itself. Then exactly the same information can be retrieve from the `deque.maxlen` property. I've added a separate commit to remove it so I can easily discard the commit if you're against this change

Reply:
I think in the future I wouldn't give access to users to the underlying `deque`, but it is nice to still provide a way to know what's the buffer size. In any case, I wouldn't decide this in this PR, we need to think about it a bit more before changing the API.

Reply:
done